ffi-cdecl/wpa_ctrl.cdef.c

```diff
@@ -1,19 +1,27 @@
+#include <errno.h>
 #include <unistd.h>
 #include <sys/un.h>
 #include <sys/socket.h>
 #include <poll.h>
+#include <sys/select.h>
```

Comment:
Is there a specific order in mind?

Reply:
None at all ^^. If this were actual C, I'd let clang-format sort it for me, FWIW ;).

Reply:
In any case, I think it'd be clearer to keep it vaguely alphabetical and such. ;-)